### DISCUSSION TITLE
Error while syft installation

### DISCUSSION THREAD

**Discussion by hashanpdn** [OpenMined/courses#398] [Course: Foundations of Private Computation]
According to the readme file I used following command to install syft.

pip install git+https://github.com/OpenMined/PySyft@dev#egg=syft

Then it returns the error: syft from git+https://github.com/OpenMined/PySyft@dev#egg=syft does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.

Please help me.

**Comment by gocandra** [comment]
it worked for me with `python3 -m pip install syft`.
the repository you are pointing to does not exist anymore, as I understand it https://github.com/OpenMined/pysyft/tree/syft_0.5.0 is what you are looking for!

**Comment by hashanpdn** [comment]
Thank you very much for your reply. I really appreciate your support.

**Reply by yemikifouly** [reply]
@hashanpdn Did the suggestion help you? If it did not, please consider posting your question in the #support channel of our Slack workspace.

Also, I realize that you had posted your enquiry several months ago. I apologize profusely for the fact that we did not attend to it earlier.
